src/Test/Utilities/Portable/Compilation/TestOperationWalker.cs

```diff
@@ -537,5 +515,22 @@ public override void VisitInvalidExpression(IInvalidExpression operation)
         {
             base.VisitInvalidExpression(operation);
         }
+
+        private void VisitArgumentsInParameterOrder(IHasArgumentsExpression operation, ImmutableArray<IParameterSymbol>? parameters)
```

Comment:
This is part of a test walker that calls every method/property in IOperation API and hoping to catch any exception that might arise. We have some #ifdef in the compiler test code that enable this for IOperation test.